npc/re/merchants/enchan_verus.txt

```diff
@@ -1154,11 +1155,6 @@ verus04,163,219,4	script	PLUTO_09#pa0829	4_SCR_AT_ROBOTS,{
 			next;
 		}
 		end;
-	default:
-		mes "^0000ffA screen showing howling that a dinosaur may not understand the command.^000000";
-		close;
-	}
-	end;
```

Comment:
Wasn't 100% sure on the reason for removing this block, could you help me understand?